dask/dataframe/io/parquet/fastparquet.py

```diff
@@ -151,7 +155,7 @@ def _determine_pf_parts(fs, paths, gather_statistics, **kwargs):
                 paths[0], open_with=fs.open, sep=fs.sep, **kwargs.get("file", {})
             )
 
-    return parts, pf, gather_statistics
+    return parts, pf, gather_statistics, fast_metadata
```

Comment:
Would be worth a comment, or even doc, as to when fast_metadata comes out True (I think: if the oath is a directory and it has a _metadata; if path is a list of files, including _metadata; otherwise, if gather_statistics is None or False)

Reply:
Good point - I added a description of the `fast_metadata` to the doc string.